arangod/Aql/HashedCollectExecutor.h

```diff
@@ -205,6 +205,8 @@ class HashedCollectExecutor {
   std::vector<Aggregator::Factory> _aggregatorFactories;
 
   GroupKeyType _nextGroupValues;
+
+  size_t _returnedGroups = 0;
```

Comment:
@goedderz: Does this variable ever need to be reset to zero, or is it fine to be ever-increasing for one HashedCollectExecutor instance?

Reply:
@jsteemann: That would be necessary in an `initializeCursor` method, if it existed. As it doesn't exist, the object will be destructed and constructed between subquery-iterations.